public/js/utils/sources-tree.js

```diff
@@ -5,8 +5,8 @@ const { isPretty } = require("../utils/source");
 const IGNORED_URLS = ["debugger eval code", "XStringBundle"];
 
 function isHiddenSource(source) {
-  const url = getURL(source);
-  return !url || url.path.match(/SOURCE/) || IGNORED_URLS.includes(url);
+  const url = source.get("url");
+  return !url || url.match(/SOURCE/) || IGNORED_URLS.includes(url);
```

Comment:
I didn't notice this before, but shouldn't we be matching on ID instead of URL anyway? We switched the id format so we could easily match `/originalSource#` I thought. That way we can change the URL to whatever we want in the future (because it determines how it's display in the UI) and none of our detection code breaks.